### DISCUSSION TITLE
Possible to disable HMR for one target?

### DISCUSSION THREAD

**Discussion by ddemaree** [parcel-bundler/parcel#8471] [Q&A]
Hello! I am using Parcel to build JS and CSS for a WordPress theme, where I have JS and CSS bundles for both the frontend (regular website styles and JS) and the WP admin (editor styles and Gutenberg block config).

HMR is super useful for iterating on my frontend styles, but it breaks the admin by unloading a bunch of unrelated styles. Is it possible to turn HMR off for only the admin, but keep it for the frontend?

**Comment by mischnic** [comment]
Not at the moment
